tests/erlang_tests/send_receive.erl

```diff
@@ -23,10 +23,44 @@
 -export([start/0, send_test/2]).
 
 start() ->
+    Test0 = test_send2(),
+    Test1 = test_send(),
+    Test2 = test_send_to_int(),
+    Test3 = test_send_to_atom(),
+    Test0 + Test1 + Test2 + Test3.
+
+send_test(V, P) ->
+    P ! V.
+
+%priv
+
+%% test send to pid
+test_send2() ->
     send_test(9, self()),
     receive
-        Value -> Value * 2
+        Value -> Value
     end.
 
-send_test(V, P) ->
-    P ! V.
+%% test send to registered process
+test_send() ->
+    erlang:register(tester, self()),
+    send_test(6, tester),
+    receive
+        RegisteredVal -> RegisteredVal
+    end.
+
+%% test send to integer
+test_send_to_int() ->
+    try 1 ! 1 of
+        _Any -> 0
+    catch
+        error:badarg -> 2
+    end.
+
+%% test send to non-registered atom
+test_send_to_atom() ->
+    try 1 ! 1 of
+        _Any -> 0
```

Comment:
Shouldn’t this be sending to a non-registered name, like `non_registered_name`?

Reply:
Yes it should. Fixed now.